src/components/form-textarea/form-textarea.js

```diff
@@ -171,13 +171,13 @@ export const BFormTextarea = /*#__PURE__*/ Vue.extend({
       const minHeight = lineHeight * this.computedMinRows + offset
 
       // Get the current style height (with `px` units)
-      const oldHeight = el.style.height || computedStyle.height
+      const oldHeight = getStyle(el, 'height') || computedStyle.height
       // Probe scrollHeight by temporarily changing the height to `auto`
-      el.style.height = 'auto'
+      setStyle(el, 'auto')
```

Comment:
Should this be `setStyle(el, 'height', 'auto')`?

Same case at line 180

Reply:
Good catch!